docker/mono-pg-dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM postgres:16
 
+# When running the `docker buildx build` command, you may encounter the following warning:
+#
+# 1 warning found (use docker --debug to expand):
+# - SecretsUsedInArgOrEnv: Do not use ARG or ENV instructions for sensitive data (ENV "POSTGRES_PASSWORD") (line 5)
+#
+# This warning indicates that sensitive data, such as passwords, should not be passed using ARG or ENV instructions
+# in the Dockerfile. This is a security best practice to prevent accidental exposure of sensitive information.
+#
+# However, since this Dockerfile is intended for use in a development environment, you can choose to ignore this warning.
```

Comment:
```suggestion
# However, since this `Dockerfile` is intended for use in a development environment, you can choose to ignore this warning, or raise a pull request to help us improving it.
```

Reply:
Minor fix